CHANGELOG

```diff
@@ -124,6 +124,14 @@ Linear and nonlinear solvers
 
 New and updated examples and miniapps
 -------------------------------------
+
+- Added new Discontinuous Petrov-Galerkin (DPG) miniapp which includes
+  serial and parallel examples for diffusion, convection-diffusion, acoustics
+  and Maxwell equations. The miniapp also includes new classes suchs as 
+  (Par)DPGWeakForm, (Par)ComplexDPGWeakForm, BlockStaticCondensation and 
+  ComplexBlockStaticCondensation. Three new integrators are added in support of 
+  DPG systems: TraceIntegrator, NormalTraceIntegrator and TangetTraceIntegrator
+
```

Comment:
This is under `Version 4.5`, and I suppose it should be moved up to line 32. Also a couple of typos: `suchs`, `Tanget`.

Reply:
fixed